ElectronClient/gui/NoteEditor/NoteBody/CodeMirror/Editor.tsx

```diff
@@ -26,6 +25,8 @@ import 'codemirror/mode/clike/clike';
 import 'codemirror/mode/diff/diff';
 import 'codemirror/mode/sql/sql';
 
+const CodeMirror = require('codemirror');
```

Comment:
This was changed by the autoformatter. I don't personally like having it below all the other imports, but it works fine.

Reply:
Any reason why this package is imported using `require()` and the other ones using `import`? Maybe the solution is to change this to use import too?

Reply:
It's an issue with how CodeMirror is set up to work with the imports. I can't import using that syntax unless we add codemirror type checking (which I will make a pull request to address soon) and add the ["esModuleInterop" flag to tsconfig.json](https://stackoverflow.com/questions/56238356/understanding-esmoduleinterop-in-tsconfig-file). 
I'm not too experienced with typescript so I didn't spend too long fussing around with it.

Reply:
yes, I think we need an exception for `ElectronClient/gui/NoteEditor/NoteBody/CodeMirror/Editor.tsx`. It currently breaks all CI builds.